parsec/interfaces/ptg/ptg-compiler/jdf2c.c

```diff
@@ -885,6 +894,15 @@ static char *dump_profiling_init(void **elem, void *arg)
                             jdf_basename, fname, 256-R, 256-G, 256-B,
                             jdf_basename, fname, jdf_basename, fname,
                             jdf_basename, fname, jdf_basename, fname);
+    // string_arena_add_string(info->sa,
```

Comment:
Do not leave a commented section like this in your code, if it's useless code, remove it before the merge.